NeuroID/NeuroIDClass/Extensions/NIDSession.swift

```diff
@@ -130,62 +86,88 @@ public extension NeuroID {
        throughout the rest of the session
       i.e. start/startSession/startAppFlow -> startAppFlow("site2") -> stop/stopSession
      */
-    static func startAppFlow(siteID: String, userID: String? = nil) -> SessionStartResult {
+    static func startAppFlow(
+        siteID: String,
+        userID: String? = nil,
+        completion: @escaping (SessionStartResult) -> Void = { _ in }
+    ) {
         if !NeuroID.verifyClientKeyExists() || !NeuroID.validateSiteID(siteID) {
-            return SessionStartResult(false, "")
-        }
+            let res = SessionStartResult(false, "")
```

Comment:
Should we consider reseting the linkedSiteId to "" here (or in the form name check) in case of a bad form name which may cause the previous name to be used in the next flow?

Reply:
That's a good callout, lets discuss post standup as a team on the behavior